RazorGenerator.Core.v1/CodeTransformers/GenericParametersTransformer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RazorGenerator.Core
+{
+    public class GenericParametersTransformer : RazorCodeTransformerBase
+    {
+        readonly string[] _parameters;
```

Comment:
@pranavkm I can certainly add if you insist but `private` is the default accessibility and isn't C# headed to allow terser expressions with each version? Why add back the obvious?